.github/schemas/source.schema.json

```diff
@@ -79,11 +79,17 @@
         },
         "runtime": {
           "type": "string"
+        },
+        "codebase": {
+          "type": "object",
+          "properties": {
+            "git-url": {
```

Comment:
1. `git-url` should be required in codebase
2. `settings` are missing (`{ "type": "array" }`)
3. `settings` should include `upgrades` (`{ "type": "array" }` including objects with `name`, `height`, `recommended-version`, `commit`, `go-version`, `ksync-engine`) and `cosmos-properties` (`{ "type": "object" }` including `registry-url`, `genesis-name`and `genesis-url`). Besides this, I would propose to set `name`, `height` and `recommended-version` as required.